framework/src/test/java/org/tron/core/actuator/AccountPermissionUpdateActuatorTest.java

```diff
@@ -205,6 +205,9 @@ private void processAndCheckInvalid(AccountPermissionUpdateActuator actuator,
       Assert.assertEquals(expectedMsg, e.getMessage());
     } catch (ContractExeException e) {
       Assert.assertFalse(e instanceof ContractExeException);
+    } catch(RuntimeException e) {
```

Comment:
If this branch is entered, this exception must be an unexcepted exception, so the second "Assert" must be a failure, I think.